exercise/templates/exercise/submission_plain.html

```diff
@@ -52,8 +60,8 @@
 					{% endif %}
 				</td>
 				<td class="points-badge">
-					<span class="hidden">{% points_badge summary %}</span>
-					{% points_badge submission %}
+					<span class="hidden">{% points_badge summary %}</span>				
+					{% points_badge submission  %}
```

Comment:
extra space?

Reply:
Made a new commit that should address the whitespace issues you pointed out.